client/network-gossip/src/state_machine.rs

```diff
@@ -23,15 +23,16 @@ use std::collections::{HashMap, HashSet};
 use std::sync::Arc;
 use std::iter;
 use std::time;
-use log::{error, trace};
+use log::{debug, error, trace};
 use lru::LruCache;
 use libp2p::PeerId;
+use prometheus_endpoint::{register, Gauge, PrometheusError, Registry, U64};
 use sp_runtime::traits::{Block as BlockT, Hash, HashFor};
 use sc_network::ObservedRole;
 use wasm_timer::Instant;
 
 // FIXME: Add additional spam/DoS attack protection: https://github.com/paritytech/substrate/issues/1115
-const KNOWN_MESSAGES_CACHE_SIZE: usize = 4096;
+const KNOWN_MESSAGES_CACHE_SIZE: usize = 8192;
```

Comment:
Maybe copy the reasoning for the concrete value from the pull request description here for future readers?